providers/ashby.go

```diff
@@ -0,0 +1,33 @@
+package providers
+
+const Ashby Provider = "ashby"
+
+func init() {
+	SetInfo(Ashby, ProviderInfo{
```

Comment:
Can we have API doc link here in comment?

Reply:
Added it.